client/signup/main.jsx

```diff
@@ -599,6 +599,7 @@ class Signup extends React.Component {
 	}
 
 	getFlowLength() {
+		flows.excludeStep( 'launch' );
```

Comment:
I can't figure out why we need this line. A code comment would be awesome.

Reply:
Good call! This change is completely unrelated to this PR. I applied it in the right one here: d29dd7c8e1a5098c872e7833407ae8668f36d2c6, added the comment here 6144e6fcc1452e2b9a0e48b5922bcc8db93caeec. I reverted this change in e36fcc5914ae975af213fe859e64c60ed4e73aa5.